package.json

```diff
@@ -5,17 +5,20 @@
   "main": "app.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "node server.js"
+    "start": "nodemon server.js"
```

Comment:
during development phase it is nodemon is better but during hosting node should be used

Reply:
okay !!!